packages/components/src/styles/components/form/super-select.scss

```diff
@@ -366,6 +366,20 @@ $hds-super-select-item-height: 36px;
       display: none;
     }
   }
+
+  .hds-form-super-select__option-group:not(:has(.ember-power-select-option[aria-selected="true"])) {
```

Comment:
For complex selectors like these, could you please add comments explaining the logic of what they are intended to do?